src/wms/src/main/java/org/geoserver/wms/svg/SVGBatikMapOutputFormat.java

```diff
@@ -98,7 +98,7 @@ private StreamingRenderer setUpRenderer(WMSMapContent mapContent) {
 
         // optimized data loading was not here, but yet it seems sensible to
         // have it...
-        Map<String, Object> rendererParams = new HashMap<String, Object>();
+        Map<Object, Object> rendererParams = new HashMap<>();
```

Comment:
Come to think about it, I guess I can make the renderer accept ``Map<?, ?>`` and avoid this change.